### PR TITLE
Update Vector examples for explicit globals

### PR DESCRIPTION
**Pull request by tildebyte**

### CODE REVIEW

File: mode/examples/Topics/Vectors/BouncingBall/BouncingBall.pyde [hunk 1]
```diff
@@ -27,7 +25,7 @@ def draw():
     # Add gravity to velocity.
     velocity.add(gravity)
     # Bounce off edges.
-    if (location.x > width) or (location.x < 0):
+    if  (location.x < 0) or (location.x > width):
```
Comment (jdf) [inline mode/examples/Topics/Vectors/BouncingBall/BouncingBall.pyde]: Might as well lose the parens.

**Comment by tildebyte** [comment]
@jdf done

File: mode/examples/Topics/Vectors/BouncingBall/BouncingBall.pyde [hunk 2]
```diff
@@ -27,7 +25,7 @@ def draw():
     # Add gravity to velocity.
     velocity.add(gravity)
     # Bounce off edges.
-    if (location.x > width) or (location.x < 0):
+    if  location.x < 0 or location.x > width:
```
Comment (jdf) [inline mode/examples/Topics/Vectors/BouncingBall/BouncingBall.pyde]: Extra space where a left paren used to be?

Comment (tildebyte) [inline mode/examples/Topics/Vectors/BouncingBall/BouncingBall.pyde]: Ayup.

**Comment by tildebyte** [comment]
@jdf Done

**Pull request merged**